modules/ztp-single-node-clusters.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-single-node-clusters_{context}"]
+
+= Single node clusters
+
+You can use ZTP to deploy single node clusters to run distributed units (DUs) on small hardware footprints at disconnected far edge sites.
+A single node cluster runs {product-title} Container Platform on top of one bare-metal machine, hence the single node.
+Single edge node servers contain a single node with supervisor functions and worker functions on the same host that are deployed at low
+bandwidth or disconnected edge sites.
+
+Hosts are partitioned to isolate the platform services, housekeeping tasks, and infrastructure pods to run on a reserved set of CPUs.
```

Comment:
sugg: "{product-title} is configured on the single node to use workload partitioning.  Workload partitioning separates cluster management workloads from user workloads and can run the cluster management workloads on a reserved set of CPUs."

My $0.02 is that "cluster management workloads" vs "user workloads" was a nice and straightforward distinction.  The "reserved CPUs" also seems important.

Reply:
Comment accepted.